Src/AutoFixture.xUnit.net2/InlineAutoDataDiscoverer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    internal class InlineAutoDataDiscoverer : DataDiscoverer
```

Comment:
Why do we need a second class that does the exact same thing as `AutoDataDiscoverer`?